backend/index.ts

```diff
@@ -118,7 +118,8 @@ if (process.env.NODE_ENV != 'test') {
       logger.info('private api listening at http://%s:%d', privateHost, privatePort)
     );
   } else {
-    privateApp.listen(privatePort, () => logger.info('private api listening at http://localhost:%d', privatePort));
+    // TODO: Next line breaks it and I can't figure out how to fix it.
+    privateApp.listen(privatePort, () => logger.info('private api listening at http://localhost%d', privatePort));
```

Comment:
🔴

Reply:
You just need to make sure your environment is set up properly. The default is not to specify a port for the private api, so it defaults to `API_PORT` + 1 (`API_PORT` itself defaults to 3000, so that would put the private API on port 3001. But the default docker config has `content-publishing-service-api` on port 3001, so there's a conflict.

Solution: add a value for `WEBHOOK_API` in the environment to use an unoccupied port

Reply:
Probably also a good idea to re-order the ports in the backend `docker-compose.yaml` so that nothing takes port 3001.

Reply:
resolved